packages/team-management/src/constants.tsx

```diff
@@ -0,0 +1,17 @@
+export const API_BASE_URL = process.env.REACT_APP_OPENSRP_API_BASE_URL;
```

Comment:
Let's not get this value from environmental variables, and continue to pass it as a prop as before because

1. Not everyone consuming the package will use environmental variables
2. The package dictates what this env should be named, forcing that the client should have this environmental var set, with exactly this name

Reply:
@KipSigei Do you have any concerns regarding this implementation or should I follow this one.

Reply:
Yes pass the URL as as a prop from the client e.g https://github.com/OpenSRP/web/blob/master/client/src/App/App.tsx#L85

Reply:
if you rebase this branch you will have the above change